SafeguardSessions/modules/schema/SchemaUtils.pqm

```diff
@@ -1,12 +1,35 @@
 let
+    IsMocked = Extension.ImportFunction("IsMocked", "Utils.pqm"),
     PaddedTable.FromRecords = Extension.ImportFunction("PaddedTable.FromRecords", "Utils.pqm"),
     SchemaTransformationErrors.ListToTextConversionError = Extension.ImportFunction(
         "ListToTextConversionError", "SchemaTransformationErrors.pqm"
     ),
+    SchemaTransformationErrors.SchemaApplicationError = Extension.ImportFunction(
+        "SchemaApplicationError", "SchemaTransformationErrors.pqm"
+    ),
     Logger.ErrorLog = Extension.ImportFunction("ErrorLog", "Logger.pqm"),
     SchemaUtils.NameColumn = "Name",
     SchemaUtils.TypeColumn = "Type",
     SchemaUtils.Header = {SchemaUtils.NameColumn, SchemaUtils.TypeColumn},
+    SchemaUtils.HandleUnexpectedSchemaApplicationError = (input as record, response as record) =>
+        if input[HasError] = true then
+            let
+                log = Logger.ErrorLog(
+                    "An error happened during the schema application to the data", [

```

Comment:
We could shorten it: "An error happened during schema apply.".
I think it holds the same meaning.

Reply:
I would prefer: "An error happened during schema application." What do you think?